pkg/virt-controller/services/template.go

```diff
@@ -1214,17 +1216,15 @@ func matchSELinuxLevelOfVMI(pod *k8sv1.Pod, vmi *v1.VirtualMachineInstance) erro
 	return nil
 }
 
-func generateContainerSecurityContext(selinuxType string, container *k8sv1.Container, forceLevel bool) {
+func generateContainerSecurityContext(selinuxType string, container *k8sv1.Container) {
```

Comment:
This should be renamed in the future imho as this is only generating the context for selinux. All other things are ignored which might be confusing as we are trying to be PSA compliant lately. I propose "forceSelinuxTypeAndLevel" and then it will be used as `forceSelinuxTypeAndLevel(container, selinuxType)`

Reply:
Agreed, and easy enough to change if I do a new revision for this PR. Otherwise I'll do it as a separate one.